staging/src/k8s.io/apiserver/pkg/reconcilers/peer_endpoint_lease.go

```diff
@@ -78,7 +78,9 @@ type peerEndpointLeaseReconciler struct {
 
 // NewPeerEndpointLeaseReconciler creates a new peer endpoint lease reconciler
 func NewPeerEndpointLeaseReconciler(config *storagebackend.ConfigForResource, baseKey string, leaseTime time.Duration) (PeerEndpointLeaseReconciler, error) {
-	leaseStorage, destroyFn, err := storagefactory.Create(*config, nil)
+	// note that newListFunc and resourcePrefix
```

Comment:
What about newFunc ?

Reply:
Same, required for a watch req, you won't receive any bookmark events without a `newFunc`. I'll update the comment.